src/features/hotspots/setup/HotspotSelectionListItem.tsx

```diff
@@ -56,6 +57,8 @@ const HotspotSelectionListItem = ({
         return <BOBCAT color={svgColor} {...svgProps} />
       case 'SYNCROBIT':
         return <SYNCROBIT color={svgColor} {...svgProps} />
+      case 'Finestra':
```

Comment:
I must have missed this one, sorry one last change this needs to be below `LONGAPONE`